### PR TITLE
updated events - Intro to RWD

### PR DESCRIPTION
**Pull request by krismyth**

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -46,6 +46,7 @@
 
                     <h3>March 2015</h3>
                     <ul>
+                        <li>3/10/15 - <a href="events/0315_rwd.html">Intro to Ruby</a></li>
```
Comment (jewlofthelotus) [inline events.html]: Hey @krismyth - the title needs to be updated here

**Comment by jewlofthelotus** [comment]
@krismyth can you reverse the domino's and optomi logos and make the optomi logo bigger so that it balances with the domino's one a bit more

**Comment by krismyth** [comment]
yes, will do

File: events.html (hunk 1 above)
Comment (krismyth) [inline events.html]: doh! will do

**Comment by jewlofthelotus** [comment]
Looks good :+1:  I made a few more design changes that you can see [here](https://github.com/gdiannarbor/gdiannarbor.com/commit/d888ddc0bfecfea5e9d3c6e3ba62cf47e61773b4) if you're interested. Merged!

**Pull request merged**